javascript/javascript1/week1/startupNames.js

```diff
@@ -0,0 +1,36 @@
+//4. Ez Namey (Startup name generator) Optional
+
+
+//First array Names:
+let firstWords = [
```

Comment:
Those arrays can also be set as `const` as you are not changing them.

Reply:
Done..